homeassistant/components/tessie/lock.py

```diff
@@ -81,18 +85,56 @@ def is_locked(self) -> bool | None:
 
     async def async_lock(self, **kwargs: Any) -> None:
         """Enable speed limit with pin."""
+        ir.async_create_issue(
+            self.coordinator.hass,
+            DOMAIN,
+            "deprecated_speed_limit_locked",
+            breaks_in_ha_version="2024.10.0",
+            is_fixable=True,
```

Comment:
Strings doesn't have a `fix_flow`

Reply:
We should have a `fix_flow`. Not change these to not fixable because a user can fix the issue (by stopping using the service).